package.json

```diff
@@ -47,19 +55,20 @@
     "url": "https://github.com/18F/analytics-reporter/issues"
   },
   "dependencies": {
-    "googleapis": "^2.1.5",
-    "node-schedule": "^0.1.13",
-    "minimist": "*",
     "async": "*",
     "aws-sdk": "*",
     "fast-csv": "*",
-    "lodash": "^3.10.1"
+    "googleapis": "^2.1.5",
+    "lodash": "^3.10.1",
+    "minimist": "*",
```

Comment:
@elisechant We'd welcome a pull request to tighten them to major version numbers or another appropriate semantic range